Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Battery/x-00-weapons.yml

```
@@ -76,19 +77,14 @@
     soundGunshot:
       path: /Audio/Weapons/Guns/Gunshots/laser3.ogg
     defaultDirection: 1, 0
-  - type: BatterySelfRecharger
-    autoRecharge: true
-    autoRechargeRate: 25
-    autoRechargePause: true
-    autoRechargePauseTime: 20
   - type: ProjectileBatteryAmmoProvider
     proto: BulletLaserHeavy
     fireCost: 30
   - type: StaticPrice
     price: 2500
 
 - type: entity
-  name: X-12 "Flamberge"
+  name: X-12 "Sorry fam, icecream machines broken"
```

actual brainrot from one of the brainrot haters lmao
```suggestion
  name: X-12 "Flamberge"